hw/ip/otp_ctrl/data/otp_ctrl.hjson.tpl

```diff
@@ -336,6 +336,69 @@
     { name: "BUS.INTEGRITY",
       desc: "End-to-end bus integrity scheme."
     }
+    { name: "SECRET.DATA.SCRAMBLE",
```

Comment:
`DATA` is currently not a possible asset. We just have `DATA_REG` and `MEM`. I think this is a case for `MEM`.

Reply:
Correction, this PR now adds `DATA` as asset. So the question becomes do we want to keep `DATA` and `MEM`? And when do we choose one over the other?

Reply:
Hm... good point. I think I did not use `MEM` in the first place since not the entire memory is encrypted, but only specific partitions are. But then again, `DATA` is very generic and may make the asset naming even more ambiguous once it is available. I'll remove it from the list and switch to `MEM` as it is more specific.